row_length: 5
rows_count: 6
row_segments:
  - scurved
row_segment_scurved_offset_min: 0.2
row_segment_scurved_offset_max: 0.5
row_segment_scurved_length_min: 5.0
row_segment_scurved_length_max: 5.0
hole_prob: [0.08, 0.08, 0.05, 0.02, 0.0, 0.0]
hole_size_max: [8, 8, 6, 4, 0, 0]
ground_resolution: 0.1
litters: 5
weeds: 5
ghost_objects: true
location_markers: true
weed_types: 
  - dandelion
